handbook/engineering/roles.md

```diff
@@ -1,55 +1,61 @@
+### Responsibilities
 
-It's impossible to estimate perfectly, so the tech lead needs to continuously monitor the team's progress and make adjustments accordingly.
+- Collaborate with [product](../product/index.md) to maintain a [prioritized roadmap](../../direction/index.md) for the work owned by your team.
+- Commit to a set of work that will be accomplished by the team for each [monthly release](releases/index.md) and capture those commitments in a tracking issue on GitHub.
+- Conduct monthly [retrospectives](retrospectives/index.md) with your team.
+- Mentor and coach engineers through regular [1-1s](../leadership/1-1.md) to level up their careers.
+- [Review the performance](../people-ops/performance-reviews.md) of your direct reports to determine compensation.
+- Interview candidates and own making improvements to our hiring process.
+- Review applications and make final hiring decisions for candidates interested in your team.
 
-If it looks like the team won't be able to complete all planned issues by the release date, the tech lead **must** communicate this in the #product channel and follow up with a combination of:
+## VP Engineering
 
-- helping unblock or accelerate the work of individual teammates
-- descoping issues in this milestone
-- deferring issues to a later milestone
-- getting help from other people on certain issues
+### Responsibilities
 
-The [Product Manager](../product/roles.md#product-manager) can help here, especially with product and priority questions for descoping or deferring work.
+TODO
 
-#### The issues for each release milestone describe the problem/solution.
+## Switching roles
 
-This means the tech lead needs to do 2 things:
+Switching between an individual contributor role and a people management role (or vice versa) is a role change (not a promotion or a demotion). Each role requires a different set of skills. Generally we expect people managers to have experience doing the role of their direct reports.
 
-- Ensure the issues for feature work have enough information for the devs who are implementing the feature.
-- Gather information for bug reports (including documenting/automating how other people can supply the info needed to diagnose/fix the bug).
+### Individual contributor to people manager
 
-## VP Engineering
+1. The IC is interested in becoming a people manager and understands how a people manager role differs from an individual contributor role.
+2. The IC's manager agrees that the IC is a good candidate for a people manager role. Ideally the IC is already doing some parts of the job that are applicable — technical leadership, planning, mentoring. The IC's manager should collect feedback from relevant teammates.
+3. The IC's manager identifies a small initial set of people for the IC to manage (e.g. 2-3 direct reports) for 1-2 months. This role change may involve, or be driven by, a team organization change.
```

Comment:
I think the deleted part ("This role change may involve, or be driven by, a team organization change.") would make more sense in the introductory paragraph at the top of the "Switching roles" section.

```suggestion
3. The IC's manager identifies a small initial set of people for the IC to manage (e.g. 2-3 direct reports) for 1-2 months.
```